docs/en_US/TrialExample/RocksdbExamples.md

```diff
@@ -39,7 +39,7 @@ In this example, the search space is specified by a `search_space.json` file as
 }
 ```
 
-*code directory: [`example/trials/systems/rocksdb-fillrandom/search_space.json`](../../../examples/trials/systems/rocksdb-fillrandom/search_space.json)*
+*code directory: [`example/trials/systems/rocksdb-fillrandom/search_space.json`](https://github.com/microsoft/nni/tree/master/examples/trials/systems/rocksdb-fillrandom/search_space.json)*
```

Comment:
When we develop in dev-branch, the link is not in the master branch. So this link `https://github.com/microsoft/nni/tree/master` might be not valid during developing.

Reply:
It is valid in `master` now. BTW, we do not know which dev branch to link to.